orx/exporter.xml

```diff
@@ -46,4 +46,31 @@
     <!-- supports texure subpath -->
     <supportsTextureSubPath>yes</supportsTextureSubPath>
 
+    <supportsPivotPoint>true</supportsPivotPoint>
+
+    <supportsAnimations>true</supportsAnimations>
+	
+	<properties>
+        <property>
+            <name>includeComments</name>
+            <type>bool</type>
+            <displayName>Include comments</displayName>
+            <toolTip>Output comments in produced result.</toolTip>
+            <isAdvanced>true</isAdvanced>
+        </property>
+        <property>
+            <name>keepInCache</name>
+            <type>bool</type>
+            <displayName>Keep in cache</displayName>
+            <toolTip>Keep textures and animations in cache.</toolTip>
+            <default>false</default>
```

Comment:
I'm not sure this is the best setting as having a per-anim/sprite `KeepInCache` value means one cannot toggle all the caching from a single place such as the default config section.

Reply:
Defaults to `false`.